### PR TITLE
Feature/dpr2 307 page controls

### PR DESCRIPTION
**Pull request by rossdowthwaite**
Version 4  - Main UI page controls

https://dsdmoj.atlassian.net/jira/software/c/projects/DPR2/boards/1282?selectedIssue=DPR2-307

### CODE REVIEW

File: app/versions/main-ui/routes.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const express = require('express')
 const router = express.Router()
 const v1Routes = require('./v1/routes')
 const v2Routes = require('./v2/routes')
+const v3Routes = require('./v4/routes')
```
Comment (stalwartsolutions) [inline app/versions/main-ui/routes.js:5]: This file looks like there's been a mis-merge. V3 and V4 have been mixed together.

File: app/versions/main-ui/v1/components/data-table-layout/views/layout.html [hunk 2]
```diff
@@ -50,7 +50,7 @@
 {% endif %}
 
   <h2 class="govuk-heading-m">
-    {% block pageTitle %}{% endblock %}
+    {% block pageTitle %}{% endblock %}csdsdsdd
```
Comment (stalwartsolutions) [inline app/versions/main-ui/v1/components/data-table-layout/views/layout.html:53]: This looks unintentional. :)

File: app/versions/main-ui/v4/routes.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+const getTodayMinusDays = days => {
+  const date = new Date()
+  date.setDate(date.getDate() - days)
+  return date.toISOString().substring(0, 10)
+}
+
+router.get('', [handlers.configureCurrentUrl, handlers.configureNavigation, function (req, res) {
+  res.render('cards', {
+    ...req.renderOptions,
+    title: 'Version 3',
```
Comment (stalwartsolutions) [inline app/versions/main-ui/v4/routes.js:29]: Version 4?

File: app/versions/main-ui/routes.js (hunk 1 above)
Comment (rossdowthwaite) [reply app/versions/main-ui/routes.js]: Nope. Just me missing places to update v3 to v4 :)

**Review by stalwartsolutions — APPROVED** [review]

**Pull request merged**